tools/topo_expl/topo_expl.cpp

```diff
@@ -158,6 +158,7 @@ NodeModelDesc model_descs[] = {
   {2, "topo_8p_940vm.xml",      "2 nodes gfx940 VM"},
   {2, "topo_8p_940_16n.xml",    "2 nodes gfx940 16 NICs"},
   {2, "topo_8p1h_6.xml",        "2 nodes 8P1H Alt."},
+  {5, "topo_8p_940.xml",        "5 nodes gfx940 8P"},
```

Comment:
Please increase https://github.com/ROCm/rccl/blob/develop/tools/scripts/topo_val.sh#L24 by one, so the script will test latest addition.